ngmix/gmix.py

```diff
@@ -684,15 +684,31 @@ def make_galsim_object(self, Tmin=0.001):
             Minimum size for gaussians.  Galsim doesn't allow objects
             with less than zero size because when convolving it renders
             the object
+        gsparams: GSParams or dict, optional
+            A GSParams object (or dict convertable to one) that sets
+            certain useful parameters for GalSim renderings (e.g.
+            a larger maximum FFT size)
         """
+
         import galsim
 
+        if gsparams and (type(gsparams) is not galsim._galsim.GSParams):
```

Comment:
should this be `if gaparams is not None and isinstance(g, galsim.GSParams)` ?

Reply:
I thought `if gsparams` and `if gsparams is not None` were functionally equivalent but I now see that this is not the case.

The second type check is actually to see if you passed something for `gsparams` that _isn't_ a `GSParams` object. That is because it tries to create a `GSParams` object if you passed it in the form of a dictionary (which sometimes happens). But I'm happy to convert this to `isinstance()` if you'd like.

Reply:
Yes, isinstance is the correct way